Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -75,42 +82,37 @@ void FullscreenManager::requestFullscreenForElement(Ref<Element>&& element, RefP
         });
     };
 
-    // 1. If any of the following conditions are true, terminate these steps and queue a task to fire
+    // If any of the following conditions are true, terminate these steps and queue a task to fire
     // an event named fullscreenerror with its bubbles attribute set to true on the context object's
     // node document:
+    if (is<HTMLDialogElement>(element)) {
```

Comment:
Probably needs a test. Seems like new behavior.

Reply:
There is a test: 
imported/w3c/web-platform-tests/fullscreen/api/element-request-fullscreen-dialog.html 

I'll rebaseline once my build finishes